cpp/bench/ann/src/cuvs/cuvs_ivf_pq.cu

```diff
@@ -17,7 +17,7 @@
 
 namespace cuvs::bench {
 template class cuvs_ivf_pq<float, int64_t>;
-// template class cuvs_ivf_pq<half, int64_t>;
+template class cuvs_ivf_pq<half, int64_t>;
```

Comment:
I assume, it's ok to keep all `cuvs_ivf_pq` in one file here, because the wrapper instances are fast to compile, right?
Do you know, why this is not the case with cagra (since we put wrapper instances for cagra in separate files)?